src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -79,7 +79,17 @@ public String AnnotationText
                 annotationModel.AnnotationText = value;                
             }
         }
-       
+
+        [JsonIgnore]
+        public string AnnotationDescriptionText
+        {
+            get { return annotationModel.AnnotationDescriptionText; }
+            set
+            {
+                annotationModel.AnnotationDescriptionText = value;
```

Comment:
'RaisePropertyChanged` not needed in setter?

Reply:
Not really here as its being raised only when the annotationModel.AnnotationDescriptionText is set, this is just a reference property it will only ever get/set models AnnotationDescriptionText so we only RaisePropertyChanged when the models prop is set